ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m

```diff
@@ -104,6 +142,10 @@ - (void)updateAttributes:(NSDictionary *)attributes
     if (attributes[@"src"]) {
         self.url = attributes[@"src"];
     }
+
+    if (attributes[@"source"]) {
```

Comment:
maybe we should update the initSource when update attributes,
as the view can be unloaded during the screen scroll, so the viewDidLoad method can called again with initSource

Reply:
done!